platform-sdk/swirlds-config-impl/src/main/java/module-info.java

```diff
@@ -7,8 +7,11 @@
     exports com.swirlds.config.impl.validators;
 
     requires transitive com.swirlds.config.api;
+    requires com.hedera.node.config;
```

Comment:
This may or may not be a good idea. Generally, the platform code is considered a library that the app/node code uses. Making the platform depend on something from the app may (in theory) create circular dependencies that may be difficult to unwind if we need to refactor the code in the future.

This may or may not be super critical for config classes. However, it would be ideal to find a way to avoid adding a dependency on the node code to the platform.

You could consider making certain configuration parameters a part of the platform config. Or, perhaps pass the values from the app to the platform via the PlatformBuilder. Or perhaps host common code in the platform common module and thus make it reusable in both the platform and the app code.

Reply:
Updated, thanks a lot for the suggestions!